pages/wiki/building-asteroidos.hbs

```diff
@@ -10,35 +10,91 @@ layout: documentation
 <div class="page-header">
   <h1 id="general-information">General information</h1>
 </div>
-These instructions have been tested on Ubuntu 19.04, but should also be applicable to Debian Sid (at least at the time of writing). For other distributions the dependencies may have different names and you may have to install additional ones.
+<p>These instructions have been tested on Ubuntu 19.04, but should also be applicable to Debian Sid (at least at the time of writing). For other distributions the dependencies may have different names and you may have to install additional ones.</p>
```

Comment:
You should move this paragraph to inside the "build without docker" section

Reply:
You should also add an introduction to why one could potentially prefer the Docker build over a build without Docker.